src/main/java/utils/Utilities.java

```diff
@@ -131,15 +136,29 @@ public static double calculateP2PDistance(
     GlobalPosition secondPoint;
     secondPoint = new GlobalPosition(lat2, lon2, 1);
     GeodeticCurve geoCurve = geoCalc.calculateGeodeticCurve(reference, firstPoint, secondPoint);
-    double ellipseKilometers = geoCurve.getEllipsoidalDistance() / (1000.0 * multiplier.getUnit());
+    double ellipseKilometers =
+        geoCurve.getEllipsoidalDistance() / (DISTANCE_RATIO_DIVIDER * multiplier.getUnit());
     geoCalc.calculateGeodeticCurve(reference, firstPoint, secondPoint);
     return ellipseKilometers;
   }
 
   public static void debug(List<Location> location) {
     for (Location point : location) {
-      System.out.println(point.getLatitude() + "," + point.getLongitude() + ";");
-      // in sout form to easy copy paste in refclient
+      log.info(point.getLatitude() + "," + point.getLongitude() + ";");
     }
   }
+
+  public static Map<String, String> getCredentials() {
+    Properties prop = new Properties();
+    ClassLoader loader = Thread.currentThread().getContextClassLoader();
+    Map<String, String> properties = new HashMap<>();
+    InputStream stream = loader.getResourceAsStream(CREDENTIALS_SOURCE);
+    try {
+      prop.load(stream);
+      properties = (Map) prop;
+    } catch (IOException e) {
+      e.printStackTrace();
```

Comment:
Please add a proper log.error messsage that states that either file does not exist or whatever the problem is